src/pages/workspaces/workspace/WorkspaceAttributeNotice.ts

```diff
@@ -48,7 +49,7 @@ const Notice = (props: NoticeProperties) => {
       h(
         TooltipTrigger,
         {
-          content: [div({ key: props.label }, [props.tooltip])],
+          content: [div({ key: props.label, style: { maxWidth: 300 } }, [props.tooltip])],
```

Comment:
Lou asked that I find a width that minimized "widows"… which I think means the tooltip text breaks in non-optimal places. This max width seemed to work pretty well for our known tooltip text.